eml/App.js

```diff
@@ -98,116 +99,31 @@ function LoginStack() {
     </Stack.Navigator>
   );
 }
-function HomeStack() {
-  checkLogin();
-
-  return (
-    <Tab.Navigator
-      initialRouteName={'Home'}
-      screenOptions={{
-        tabBarActiveTintColor: 'black',
-        tabBarActiveBackgroundColor: '#d9d9d9',
-        tabBarStyle: { backgroundColor: 'hsl(0, 0%, 92%)' }, //Oneplus menubar color
-      }}
-    >
-      <Tab.Screen
-        //Home
-        name="Casa"
-        component={CourseStack}
-        options={{
-          headerShown: false,
-          tabBarIcon: () => {
-            return (
-              <Icon
-                size={30}
-                name="home"
-                type="material-community"
-                color="#8DD08C"
-              />
-            );
-          },
-        }}
-      />
-      <Tab.Screen
-        //Perfil
-        name="Perfil"
-        component={ProfileComponent}
-        options={{
-          headerShown: false,
-          tabBarIcon: () => {
-            return (
-              <Icon
-                size={30}
-                name="account-circle"
-                type="material-community"
-                color="#8DD08C"
-              />
-            );
-          },
-        }}
-      />
-      <Tab.Screen
-      // Explore
-        name="Explorar"
-        component={Explore}
-        options={{
-          headerShown: false,
-          tabBarIcon: () => {
-            return (
-              <Icon
-                size={30}
-                name="magnify"
-                type="material-community"
-                color="#8DD08C"
-              />
-            );
-          },
-        }}
-      />
-      {/*       <Tab.Screen
-        name="TestScreen"
-        component={TestScreen}
-        options={{
-          headerShown: false,
-          tabBarIcon: () => {
-            return (
-              <Icon
-                size={30}
-                name="home"
-                type="material-community"
-                color="black"
-              />
-            );
-          },
-        }}
-      /> */}
-    </Tab.Navigator>
-  );
-}
```

Comment:
What is the reason for deleting the whole stack?

Reply:
Because we made a 'NavBar' component instead.